tests/test_visitors/test_ast/test_operators/test_bitwise_boolean_mixup.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    BitwiseAndBooleanMixupViolation,
+)
+from wemake_python_styleguide.visitors.ast.operators import BitwiseOpVisitor
+
+# add more cases for more usages of mixup
+
+
+@pytest.mark.parametrize('expression', [
+    'True | False',
+    '(x >= y) & True',
+    'first | (10 == second)',
+    '(11 != first) | (not False)',
+    '(1 or first) & (second and first)',
+    '(first > 0 & second != 0) | (other is None)',
+    '(5234 + 2345) & True',
```

Comment:
What about `None`? Should it be counted?

Reply:
I think it should be counted as a violation (python will  throw an error when NoneType is used with Bitwise operators anyways).  Added `None` test case.